### ISSUE TITLE
Please add Revolt heating thermostat NX-4911-919

### ISSUE THREAD

**Issue by Jios93** [ioBroker/ioBroker.zigbee#837]
Please add Revolt heating thermostat 
NX-4911-919

https://www.revolt-power.de/Weltweit-steuerbares-Heizkoerper-Design-Thermostat-NX-4911-919.shtml

**Comment by datwusel** [comment]
Hello everybody,

I'm a noob in programing but I started with the guidance "how to support new devices".

I've got some readings from the mentioned revolt device but I don't know how to create the required converters for fromZigbee and toZigbee.

Readings when changing temperature to 17.5 degrees:

Zigbee2MQTT:debug 2020-11-07 14:59:13: Received Zigbee message from '0xbc33acfffe656eba', type 'commandGetData', cluster 'manuSpecificTuyaDimmer', data '{"data":{"data":[0,0,0,175],"type":"Buffer"},"dp":514,"fn":0,"status":0,"transid":36}' from endpoint 1 with groupID 0

Zigbee2MQTT:debug 2020-11-07 14:59:13: No converter available for 'NX-4911_thermostat' with cluster 'manuSpecificTuyaDimmer' and type 'commandGetData' and data '{"data":{"data":[0,0,0,175],"type":"Buffer"},"dp":514,"fn":0,"status":0,"transid":36}'

Thermostat has open window detection and time schedule.

Is someone here who can help me?

Cheers
